Assets/Creatures/api/Genera.cs

```diff
@@ -0,0 +1,9 @@
+using UnityEngine;
+
+namespace Creatures.Api
+{
+    public interface Genera
+    {
+        string GeneraType { get; }
```

Comment:
another thought: should this be an enum, or do we want this to be that dynamic?

Reply:
fixt

Reply:
good call made the change